lib/u3d/installation.rb

```diff
@@ -44,6 +44,39 @@ def self.create(root_path: nil, path: nil)
         WindowsInstallation.new(root_path: root_path, path: path)
       end
     end
+
+    def self.package_aliases
+      {
+        'Android' => [],
+        'iOS' => ['iPhone'],
+        'AppleTV' => ['tvOS'],
+        'Linux' => ['StandaloneLinux'],
+        'Mac' => %w[StandaloneOSXIntel StandaloneOSXIntel64 StandaloneOSX],
+        'Windows' => ['StandaloneWindows'],
+        'Metro' => [],
+        'UWP-IL2CPP' => [],
+        'Samsung-TV' => [],
+        'Tizen' => [],
+        'WebGL' => [],
+        'Facebook-Games' => ['Facebook'],
+        'Vuforia-AR' => ['UnityExtensions']
+      }
+    end
+
+    def packages
+      false
+    end
+
+    def package_installed?(package)
+      return false unless packages
```

Comment:
we can replace the two lines with

```
return true if (packages || []).include?(package)
```

and maybe change the default `packages` implementation to return a `[]` instead of false?